src/framework/vst/view/vstfxeditorview.cpp

```diff
@@ -0,0 +1,140 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "vstfxeditorview.h"
+
+#include <QWindow>
+#include <QResizeEvent>
+
+#include "async/async.h"
+
+#include "vsttypes.h"
+#include "internal/vstplugin.h"
+
+using namespace mu::vst;
+using namespace Steinberg;
+
+IMPLEMENT_FUNKNOWN_METHODS(VstFxEditorView, IPlugFrame, IPlugFrame::iid)
+
+VstFxEditorView::VstFxEditorView(QWidget* parent)
+    : QDialog(parent)
+{
+    setAttribute(Qt::WA_NativeWindow, true);
+}
+
+VstFxEditorView::VstFxEditorView(const VstFxEditorView& copy)
+    : VstFxEditorView(copy.parentWidget())
+{
+}
+
+VstFxEditorView::~VstFxEditorView()
+{
+    if (m_view) {
+        m_view->removed();
+    }
+}
+
+tresult VstFxEditorView::resizeView(IPlugView* view, ViewRect* newSize)
+{
+    setGeometry(QRect(geometry().x(), geometry().y(), newSize->getWidth(), newSize->getHeight()));
+    view->onSize(newSize);
+
+    return kResultTrue;
+}
+
+void VstFxEditorView::wrapPluginView()
+{
+    VstPluginPtr plugin;
+
+    if (m_trackId == -1) {
+        plugin = pluginsRegister()->masterFxPlugin(m_resourceId.toStdString());
+    } else {
+        plugin = pluginsRegister()->fxPlugin(m_trackId, m_resourceId.toStdString());
+    }
+
+    if (!plugin || !plugin->view()) {
+        return;
+    }
+
+    m_view = plugin->view();
+
+    if (m_view->isPlatformTypeSupported(currentPlatformUiType()) != Steinberg::kResultTrue) {
+        return;
+    }
+
+    m_view->setFrame(this);
+
+    Steinberg::tresult attached;
+    attached = m_view->attached(reinterpret_cast<void*>(windowHandle()->winId()), currentPlatformUiType());
+    if (attached != kResultOk) {
```

Comment:
You can just remove this if (or add here some kind of error checking)

Reply:
Recently there was an explicit call of "resize" after that condition
Now we don't need this probably

Reply:
added LOGE() console output